charts/openstack-actions-runner/Chart.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: v2
+apiVersion: v1
```

Comment:
I would recommend just removing this field. (if that works)

That's our bad but currently we use helm2 in releng-it and you are using helm3 (and you should). But this chart needs to be compatible with both for now.

Reply:
It don't work without the `apiVersion`